src/gui/element/catalog.rs

```diff
@@ -276,7 +278,7 @@ pub fn data_row_container<'a, 'b>(
             .game_versions
             .iter()
             .find(|gv| gv.flavor == config.wow.flavor.base_flavor())
-            .map(|gv| gv.game_version.clone())
+            .map(|gv| format_interface_into_game_version(&gv.game_version[..]))
```

Comment:
All other backends have a properly formatted game version that isn't 5 characters? Just want to make sure this couldn't ever impact data from the other backends?

Reply:
I've added test to at least ensure it's as I expect. But don't know if it's prettier we do it earlier in the deser progress

Reply:
I think it's fine to leave it here... maybe just add logic to only evaluate that mapping for TownlongYak source? `if addon_data.source == TownlongYak { format_interface_into_game_version(&gv.game_version[..]) } else { gv.game_version.clone() }`

Reply:
I think that make sense to do. Ill have to go to bed so the RC has to wait. If you dont add it ill do it tomorrow tomorrow :-)

Reply:
Sounds good, have a good night!